contents/manual/group-analytics.mdx

```diff
@@ -206,6 +206,33 @@ import GroupsFeatureFlagsNode from './snippets/groups-flags-node.mdx'
 
 You can change how group types are displayed in your Insights interface and throughout PostHog on Project settings.
 
+## Naming groups
```

Comment:
Looking at the whole doc, this is positioned weirdly - just after feature flags and settings.

Let's add this as the last `###` under `Viewing groups and their properties`?